samza-sql/src/main/java/org/apache/samza/sql/util/SamzaSqlQueryParser.java

```diff
@@ -117,7 +113,8 @@ public static QueryInfo parseQuery(String sql) {
       sink = sqlInsert.getTargetTable().toString();
       if (sqlInsert.getSource() instanceof SqlSelect) {
         SqlSelect sqlSelect = (SqlSelect) sqlInsert.getSource();
-        selectQuery = m.group(2);
+        selectQuery = sqlSelect.toString();
+        LOG.info("Parsed select query {} from sql {}", selectQuery, sql);
         sources = getSourcesFromSelectQuery(sqlSelect);
       } else {
         throw new SamzaException("Sql query is not of the expected format");
```

Comment:
nit: not directly related to this PR, but might want to add what the expected format is in the exception message here and below.